front/style/layout/_buttons-site.scss

```diff
@@ -0,0 +1,23 @@
+@mixin btn--as-link {
+    background: none!important;
```

Comment:
Did you try just removing the `important!` tags in this file instead of creating a new file just for amp buttons? What happened when you did that?

Reply:
I was thinking that buttons I wouldn't break something because there are used every where :s

Reply:
Cool, let's just remove the `important` tags then :)